core/kazoo_services/src/kz_services.erl

```diff
@@ -573,7 +573,7 @@ public_json(#kz_services{jobj=ServicesJObj
                          ,cascade_quantities=CascadeQuantities
                         }) ->
     AccountId = kz_doc:account_id(ServicesJObj),
-    InGoodStanding = try maybe_follow_billling_id(AccountId, ServicesJObj) of
+    InGoodStanding = try maybe_follow_billing_id(AccountId, ServicesJObj) of
```

Comment:
BTW these typos have already been fixed. Changes to this file should be reverted (or PR rebased)